ibis/tests/sql/test_compiler.py

```diff
@@ -2221,6 +2221,27 @@ def test_endswith(self):
         assert Compiler.to_sql(expr) == expected
 
 
+QUERY = """\
+SELECT `string_col` AS `key`, CAST(`float_col` AS double) AS `value`
+FROM functional_alltypes
+WHERE `int_col` > 0
+{}
+SELECT `string_col` AS `key`, `double_col` AS `value`
+FROM functional_alltypes
+WHERE `int_col` <= 0"""
+
+SELECT_QUERY = """SELECT `key`
+FROM (
+  SELECT `string_col` AS `key`, CAST(`float_col` AS double) AS `value`
+  FROM functional_alltypes
+  WHERE `int_col` > 0
+  {}
+  SELECT `string_col` AS `key`, `double_col` AS `value`
+  FROM functional_alltypes
+  WHERE `int_col` <= 0
+) t0"""
```

Comment:
Still a significant amount of duplication, wouldn't make sense to use inside the test something like this or similar instead?
```
SELECT `key` FROM ({QUERY.format('INTERSECT')}) t0
```

Reply:
It's either that or something like this

```python
QUERY = """\
SELECT `string_col` AS `key`, CAST(`float_col` AS double) AS `value`
FROM functional_alltypes
WHERE `int_col` > 0
{}
SELECT `string_col` AS `key`, `double_col` AS `value`
FROM functional_alltypes
WHERE `int_col` <= 0"""

SELECT_QUERY = f"SELECT `key` FROM ({QUERY}) t0"
```

I would just need to some add new the two spaces in front of each new line `QUERY`. Maybe your suggestions is nicer?

Reply:
Ah, true. i think it's still worth, you can use `textwrap.indent`. But up to you, no big deal.

Reply:
Haha should have known there is a package for that. Done!